src/Tests/Microsoft.Diagnostics.Monitoring.Tool.UnitTests/LogsTests.cs

```diff
@@ -44,14 +44,14 @@ public LogsTests(ITestOutputHelper outputHelper, ServiceProviderFixture serviceP
         /// Tests that all log events are collected if log level set to Trace.
         /// </summary>
         [Theory]
-        [InlineData(DiagnosticPortConnectionMode.Connect, LogFormat.JsonSequence)]
-        [InlineData(DiagnosticPortConnectionMode.Connect, LogFormat.NDJson)]
+        [InlineData(Options.DiagnosticPortConnectionMode.Connect, LogFormat.JsonSequence)]
```

Comment:
Could you use `using Microsoft.Diagnostics.Monitoring.UnitTests.Options;` instead of clarifying the namespace for each of these? Even if there is ambiguity, it's probably a smaller subset to change the other types to be clarified or to use a type using e.g. `using Baz = Foo.Bar.Baz;`

Reply:
I believe for these I did it since LogFormat was in WebApi (so I had to allow WebApi to be included); I can change it to whatever you'd prefer though